pulsar-functions/worker/src/main/java/org/apache/pulsar/functions/worker/SchedulerManager.java

```diff
@@ -105,25 +105,12 @@ public SchedulerManager(WorkerConfig workerConfig, PulsarClient pulsarClient, Pu
                     .compressionType(CompressionType.LZ4)
                     .sendTimeout(0, TimeUnit.MILLISECONDS)
                     .createAsync().get(10, TimeUnit.SECONDS);
+            } catch (Exception e) {
```

Comment:
I don't think we should retry at `InterruptedException` and also when `InterruptedException` is caught, you should propagate the interrupted bit to your outer thread `Thread.currentThread().interrupt()`

Reply:
@sijie I've fixed the handling of the interrupted exception